src/test/java/MathematicsTest.java

```diff
@@ -0,0 +1,63 @@
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertTrue;
+
+/**
+ * Created by LiudaBalatskaya on 5/8/2017.
+ */
+public class MathematicsTest {
+    public int result;
+    public Mathematics calculator;
+
+    @BeforeClass
+    public static void beforeClass(){
+        Mathematics.logger.info("In beforeClass method");
+    }
+    @BeforeMethod
+    public void initializerTest() {
+        calculator = new Mathematics();
+        calculator.setResult(0);
+    }
+    @AfterClass
+    public static void finishedClass(){
+        Mathematics.logger.info("Testing is finished");
+    }
+    @Test(groups = {"smoke", "fast"})
+    public void addCheck() {
+        calculator.add(2,2);
+        result = calculator.getResult();
+        assertTrue(result==4);
```

Comment:
Do not use assertTrue in this case. In case of fail you will see message like "Expected true but found false". In case of assertEquals you will see message: "Expected 4 but found  5" - that much more handy to debugging.
4 is hardcode - create separate variable